schema/workflow.json

```diff
@@ -203,6 +203,24 @@
     "states"
   ],
   "definitions": {
+    "crondef": {
+      "type": "object",
+      "properties": {
+        "expression": {
+          "type": "string",
+          "description": "Repeating interval (cron expression) describing when the workflow instance should be created",
+          "minLength": 1
+        },
+        "deadline": {
```

Comment:
Nit; I prefer a name like 'valid-until' as deadline seems a bit ambiguous to me.

Reply:
We lend the term from k8s: https://kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/#starting-deadline. I understand that it might not relate, and I liked `valid-until` better. @tsurdilo wdyt?

Reply:
@jorgenj @ricardozanini would "validUntil" be ok? That would fit our camel-case parameter names used everywhere else in lang.

Reply:
updated to validUntil :)